handlers.go

```diff
@@ -257,30 +217,37 @@ func runQuery(db dbType,
 	}
 
 	// ensure rows gets closed at end
-	defer func() {
-		_ = rows.Close()
-	}()
+	defer rows.Close()	// nolint
 
 	cols, err := rows.Columns() // Remember to check err afterwards
 	if err != nil {
 		return ret, err
 	}
 	log.Debugf("cols = %s", cols)
+	ncols := len(cols)
+
+	count := 0
 
 	for rows.Next() {
 		rownum ++
```

Comment:
You don't need `rownum` neither, youcan just use `len(ret)` wherever you want `rownum` or `count`